packages/design-tokens/docs/StoryBoard.stories.tsx

```diff
@@ -140,9 +141,38 @@ const ColorDemo = (props: { color: string; name?: string }) => {
           width: "20rem",
           height: "10rem",
           backgroundColor: props.color,
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "center",
+          alignItems: "center",
           ...theme.border.solid,
         }}
-      ></div>
+      >
+        {parsedColor && (
+          <Heading variant="heading-5">
+            <span
+              style={{
+                color: props.color,
+                filter: "invert(1) grayscale(1) contrast(100)",
```

Comment:
This is clever. 

Back on the old Kaizen site I had [a whole contrast calculation thing](https://github.com/cultureamp/cultureamp-style-guide/blob/master/guide/src/pages/styles/colors/_ColorBlock.js#L18) to determine if text should be white or black. If you did something like that you could use the Heading's actual props to toggle colour, rather than this span with inline styles.

Not sure it's worth it though.

Reply:
Yeah this is just to potentially help Marc, so probably not worth worrying about too much, and as you say, the crossover between storybook and gatsby will need to be cleared up - we'll probably ditch this color demo at some point maybe.